terraform/modules/iot-edge/main.tf

```diff
@@ -31,15 +31,27 @@ resource "azurerm_public_ip" "iot_edge" {
 }
 
 resource "azurerm_network_security_group" "iot_edge" {
-  name                = "nsg-${local.dns_label_prefix}"
+  name                = "nsg-iotedge-${var.resource_uid}"
   resource_group_name = var.resource_group_name
   location            = var.location
 
   security_rule {
-    name                       = "nsgr-allow-22"
-    priority                   = 1000
+    name                       = "AllowBastionInbound"
```

Comment:
Let's use the CAF recommended abbreviation `nsgsr-` (I incorrectly put it as `nsgr` before)